### PR TITLE
Add implementations of the Octree trait in a new gRPC crate.

### PR DESCRIPTION
**Pull request by SirVer**
Related to #53.

### CODE REVIEW

File: ci/script.sh [hunk 1]
```diff
@@ -1,18 +1,27 @@
...
 main() {
+    # root
     cargo build --verbose
     cargo test
 
+    # grpc
+    pushd grpc
+    cargo build --verbose
+    cargo test
+
```
Comment (thomasschiwietz) [inline ci/script.sh]: missing popd

File: ci/install.sh [hunk 2]
```diff
@@ -20,6 +20,7 @@ main() {
     export PATH="$PATH:$HOME/bin"
 
     cargo install --force protobuf
+    cargo install --force grpcio-compiler
```
Comment (thomasschiwietz) [inline ci/install.sh]: add comment in README

**Comment by thomasschiwietz** [comment]
LGTM

**Comment by SirVer** [comment]
@wally-the-cartographer merge

**Comment by wally-the-cartographer** [comment]
Merge requested by authorized user SirVer. Merge queue now has a length of 1.

**Comment by wally-the-cartographer** [comment]
Error: PUT https://api.github.com/repos/googlecartographer/point_cloud_viewer/pulls/68/merge: 405 1 review requesting changes by reviewers with write access. []

Log:
~~~
=> handle work item: PR 68 on googlecartographer/point_cloud_viewer.
 => Switching to branch master.
 $ git reset --hard HEAD [data/point_cloud_viewer]
HEAD is now at bb73289 Make beta required to pass on travis again. (#67)
 $ git clean -fd [data/point_cloud_viewer]
 $ git checkout master [data/point_cloud_viewer]
Already on 'master'
Your branch is up to date with 'origin/master'.
 $ git pull [data/point_cloud_viewer]
Already up to date.
Current branch master is up to date.
 => Checking out SirVer/02_add_grpc_crate as local branch pr_68.
 $ git remote add SirVer git@github.com:SirVer/point_cloud_viewer.git [data/point_cloud_viewer]
fatal: remote SirVer already exists.
 $ git fetch SirVer [data/point_cloud_viewer]
 $ git branch --track pr_68 SirVer/02_add_grpc_crate [data/point_cloud_viewer]
Branch 'pr_68' set up to track remote branch '02_add_grpc_crate' from 'SirVer' by rebasing.
 => Switching to branch pr_68.
 $ git reset --hard HEAD [data/point_cloud_viewer]
HEAD is now at bb73289 Make beta required to pass on travis again. (#67)
 $ git clean -fd [data/point_cloud_viewer]
 $ git checkout pr_68 [data/point_cloud_viewer]
Switched to branch 'pr_68'
Your branch is up to date with 'SirVer/02_add_grpc_crate'.
 $ git pull --force [data/point_cloud_viewer]
Already up to date.
Current branch pr_68 is up to date.
 => merge master.
 $ git merge master [data/point_cloud_viewer]
Already up to date.
 => Running clang-format on all *.cc and *.h files
 $ git commit -am Ran clang-format. [data/point_cloud_viewer]
On branch pr_68
Your branch is up to date with 'SirVer/02_add_grpc_crate'.

nothing to commit, working tree clean
 $ git push --force SirVer HEAD:02_add_grpc_crate [data/point_cloud_viewer]
Everything up-to-date
 => Abandoning branch pr_68.
 => Switching to branch master.
 $ git reset --hard HEAD [data/point_cloud_viewer]
HEAD is now at 1cf5b85 Fixed README.md.
 $ git clean -fd [data/point_cloud_viewer]
 $ git checkout master [data/point_cloud_viewer]
Switched to branch 'master'
Your branch is up to date with 'origin/master'.
 $ git branch -D pr_68 [data/point_cloud_viewer]
Deleted branch pr_68 (was 1cf5b85).

~~~

**Comment by SirVer** [comment]
@wally-the-cartographer merge

**Comment by wally-the-cartographer** [comment]
Merge requested by authorized user SirVer. Merge queue now has a length of 1.

**Pull request merged**